conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -22,11 +22,11 @@ fi
 source ${HOME}/miniforge3/etc/profile.d/conda.sh
 conda activate base
 
-echo -e "\n\nInstalling conda-forge-ci-setup=3 and conda-build."
-conda install -n base --quiet --yes conda-forge-ci-setup=3 conda-build pip
+echo -e "\n\nInstalling {{ remote_ci_setup }} and conda-build."
+conda install -n base --quiet --yes {{ remote_ci_setup }} conda-build pip
 
 {% if local_ci_setup %}
-conda uninstall --quiet --yes --force conda-forge-ci-setup
+conda uninstall --quiet --yes --force {{ remote_ci_setup }}
```

Comment:
Does this command work if there is a channel alias in `remote_ci_setup`?

Reply:
Seems to for me:

```shell
conda uninstall --quiet --yes --force conda-forge::conda-forge-ci-setup=3

## Package Plan ##

  environment location: /home/tom/conda-project/miniforge3/envs/deleteme

  removed specs:
    - conda-forge::conda-forge-ci-setup=3


The following packages will be REMOVED:

  conda-forge-ci-setup-3.2.1-py38h3611fdf_0


Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
```

Reply:
awesome!

Reply:
fwiw it seems to work for fully qualified channel urls too

```shell
conda uninstall http://conda.anaconda.org/conda-forge::conda-forge-ci-setup=3
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/tom/conda-project/miniforge3/envs/deleteme

  removed specs:
    - conda-forge::conda-forge-ci-setup=3
```